recipes-tag/py4xs/meta.yaml

```diff
@@ -1,14 +1,14 @@
-{% set version = "2017.12.20" %}
+{% set version = "2019.6.28.0" %}
 {% set name = "py4xs" %}
 
 package:
     name: {{ name }}
     version: {{ version }}
 
+
```

Comment:
```suggestion
```

Reply:
I manually applied the changes and built the package locally within a `nsls2/debian-with-miniconda` container, it built fine.